static/app/views/organizationGroupDetails/groupReplays/groupReplays.tsx

```diff
@@ -0,0 +1,148 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import Link from 'sentry/components/links/link';
+import Pagination from 'sentry/components/pagination';
+import {PanelTable} from 'sentry/components/panels';
+import {IconArrow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {PageContent} from 'sentry/styles/organization';
+import {Group, NewQuery} from 'sentry/types';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {getQueryParamAsString} from 'sentry/utils/replays/getQueryParamAsString';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
+import ReplayTable from 'sentry/views/replays/replayTable';
+import {Replay} from 'sentry/views/replays/types';
+
+const DEFAULT_DISCOVER_LIMIT = 50;
+
+type Props = {
+  group: Group;
+};
+
+const GroupReplays = ({group}: Props) => {
+  const location = useLocation();
+  const organization = useOrganization();
+  const params = useParams();
+  const {
+    project: {id},
+  } = group;
+
+  const getEventView = () => {
+    const {groupId} = params;
+    const eventQueryParams: NewQuery = {
+      id: '',
+      name: '',
+      version: 2,
+      fields: [
+        'eventID',
+        'project',
+        'timestamp',
+        'url',
+        'user.display',
+        'user.email',
+        'user.id',
+        'user.ip_address',
+        'user.name',
+        'user.username',
+      ],
+      projects: [+id],
```

Comment:
nit, fully destructuring this makes `id` a bit vague, I would just destructure `project` to make this `project.id`